src/se.jbee.inject/main/java/se/jbee/inject/Generator.java

```diff
@@ -29,6 +29,6 @@
 	 *             to create the requested instance because another instance
 	 *             needed to create it could not be resolved.
 	 */
-	T yield(Dependency<? super T> dep) throws UnresolvableDependency;
+	T yielt(Dependency<? super T> dep) throws UnresolvableDependency;
```

Comment:
It is `yield`. Also try to avoid PRs with mixed changes (renames, behaviour changes, organisation) - especially when they get large.

Reply:
`yield` is a keyword since Java 13; https://openjdk.java.net/jeps/361